packages/owl-bot/src/config-files.ts

```diff
@@ -40,3 +43,33 @@ export function owlBotLockFrom(o: Record<string, any>): OwlBotLock {
   }
   return o as OwlBotLock;
 }
+
+export interface CopyDir {
+  source: string;
+  dest: string;
+}
+
+// The .github/.OwlBot.yaml is stored on each repository that OwlBot
+// is configured for, and indicates the docker container that should be run
+// for post processing and which files from googleapis-gen should be copied.
+export interface OwlBotYaml {
+  docker: {
+    image: string;
+  };
+  'copy-dirs': CopyDir[];
```

Comment:
I try to avoid property names with dashes like this because they're a pain to deal with in JavaScript.  I wonder if ajv will do auto kebab-camel conversion, like meow does?